be/src/exec/es/es_predicate.cpp

```diff
@@ -49,6 +49,14 @@ namespace doris {
 
 using namespace std;
 
+#define RETURN_ERROR_IF_EXPR_IS_NOT_SLOTREF(expr) \
+    do { \
+            const Expr* expr_without_cast = Expr::expr_without_cast(expr); \
+            if (expr_without_cast->node_type() != TExprNodeType::SLOT_REF) { \
+                return Status::InternalError("build disjuncts failed: child is not slot ref"); \
```

Comment:
what is slot ref? add some example otherwise  it will be very confusing once this error occurs.
 e.g. slot ref(xx, xxx)